packages/react-scripts/utils/createJestConfig.js

```diff
@@ -12,11 +12,18 @@
 const fs = require('fs');
 const paths = require('../config/paths');
 
-module.exports = (resolve, rootDir, isEjecting) => {
+module.exports = (opts) => {
+  const resolve = opts.resolve;
+  const rootDir = opts.rootDir || null;
+  const isEjecting = opts.isEjecting || false;
+  const isDebug = opts.isDebug || false;
+
   // Use this instead of `paths.testsSetup` to avoid putting
   // an absolute filename into configuration after ejecting.
   const setupTestsFile = fs.existsSync(paths.testsSetup) ? '<rootDir>/src/setupTests.js' : undefined;
-
+  const babelTransform = isEjecting
+    ? '<rootDir>/node_modules/babel-jest'
+    : resolve('config/jest/babelTransform' + (isDebug ? 'Debug' : '') + '.js')
```

Comment:
Ok I hadn't reviewed deep enough to have this understanding.
Thanks for explaining!

I'll snoop around and see what our best option(s) are. 😄